### ISSUE TITLE
[Supporting the Platform- Botkit]: Unable to identify the selected links i.e. “Get started, Docs, Support and Blog” in High contrast Black and White Modes.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#14]
**User Experience:**
Low vision people unable to identify the selected links if selected links are not visible then the low vision users will be affected. 

**Test Environment:**
Url: https://botkit.ai/
OS: Version 1809(OS Build 17763.379)
Browser: Edge
Ease of Access: Settings->High contrast modes

**Repro Steps:** 
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to “Get started, Docs, Blog and Support” links 
Step 3: Apply the high contrast Black and White Modes. 
Step 4: Observe the issue.

**Actual Result:** 
Unable to identify the selected links i.e. “Get started, Docs, Support and Blog” in High contrast Black and White Modes.
 
**Expected Result:** 
User should be identifying the selected links i.e. “Get started, Docs, Support and Blog” in High contrast Black and White Modes.
There should be some difference between the selected link and Not Selected link.
 
**MAS Reference:** 
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}](url)

![MAS4 3 1_BotKit_Image Links](https://user-images.githubusercontent.com/48272005/54514020-01b7ae80-497f-11e9-86e5-87910676a7fd.png)

**Comment by benbrown** [comment]
I have changed the way this is implemented so that the selected link should be highlighted and the other links should not.

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
Now able to identify the selected links i.e. “Get started, Docs, Support and Blog” in High contrast Black and White Modes.
